src/journal-remote/journal-upload.c

```diff
@@ -562,6 +571,7 @@ static int parse_config(void) {
                 { "Upload",  "ServerKeyFile",          config_parse_path_or_ignore, 0, &arg_key    },
                 { "Upload",  "ServerCertificateFile",  config_parse_path_or_ignore, 0, &arg_cert   },
                 { "Upload",  "TrustedCertificateFile", config_parse_path_or_ignore, 0, &arg_trust  },
+                { "Upload",  "CurlTimeoutSecs",        config_parse_long,   0, &arg_curl_timeout_secs },
```

Comment:
I think the option should be named `NetworkTimeoutSec` (even if the internal field is called `arg_network_usec`, i.e. internally we should always deal with µs even if the default time unit we expose is just s. config_parse_usec() handles that for you.

Reply:
also, please realign the table to ensure it remains nicely aligned

Reply:
also, needs to be documented in the man page

Reply:
> I think the option should be named `NetworkTimeoutSec` (even if the internal field is called `arg_network_usec`, i.e. internally we should always deal with µs even if the default time unit we expose is just s. config_parse_usec() handles that for you.

I think we don't have a config_parse_usec. As per the use of the macro DEFINE_PARSER in src/shared/conf-parser.c, there is a config_parse_nsec (uses parse_nsec) and config_parse_sec (uses parse_sec) .  So.. should I use an alternative unit or... ?

Reply:
config_parse_sec() is what you want to use. It parses into an usec_t and can parse with µs granularity if you explicitly specify µs as unit. But if you just provide a unit-less integer it assumes seconds, hence the name. Yeah, it's a bit confusing: default unit and granularity are distinct here.